packages/typespec-ts/src/index.ts

```diff
@@ -222,24 +222,28 @@ export async function $onEmit(context: EmitContext) {
     }
     const rlcClient: RLCModel = rlcCodeModels[0];
     const option = dpgContext.rlcOptions!;
+    const isBranded = option.branded ?? true;
```

Comment:
I was hoping we could calculate this from the lack of Azure.* dependencies in the TypeSpec instead of it being a flag

Reply:
I wouldn't block this PR on this, since probably we need to come up with an agreement cross language about how to detect this. but lets file an issue

Reply:
I filed an issue https://github.com/Azure/autorest.typescript/issues/2101, but my understanding is that given typespec we could generate both Azure SDK and Non-Azure SDK as output. If we calculate them automatically how to implement that?

Reply:
Currently, dotnet SDK is also using a flag to branding/unbranding.  So, I think it is fine for now to quickly unblock the demo, but we need to refactor it later.